### PR TITLE
slides 19

### PR DESCRIPTION
**Pull request by rajathkmp**

### CODE REVIEW

**Comment by amueller** [comment]
always send PRs from a feature branch ;)

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 1]
```diff
@@ -0,0 +1,1020 @@
...
+lr.C_
+```
+```
+array([ 0.046])
+```
+```python
+lr.score(X_val, y_val)
+```
+```
+0.88192000000000004
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html]: please round to three decimals (always)

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 2]
```diff
@@ -0,0 +1,1020 @@
...
+lr.C_
+```
+```
+array([ 0.046])
+```
+```python
+lr.score(X_val_df4, y_val)
+```
+```
+0.88095999999999997
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html]: round to three decimals

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 3]
```diff
@@ -0,0 +1,1020 @@
...
+```
+```python
+vect3msw = CountVectorizer(ngram_range=(1, 3), min_df=4, stop_words=my_stopwords)
+X_train3msw = vect3msw.fit_transform(text_train_sub)
+lr3msw = LogisticRegressionCV().fit(X_train3msw, y_train_sub)
+X_val3msw = vect3msw.transform(text_val)
+lr3msw.score(X_val3msw, y_val)
+```
+```
+0.88319999999999999
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html]: round

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 4]
```diff
@@ -0,0 +1,1020 @@
...
+.center[
+![:scale 90%](images/stopwords_2.png)
+]
+---
+
+# Tf-idf rescaling
+
+$$ tf\text{-}idf(t,d) = tf(t,d)\cdot idf(t)$$
+
+$$ idf(t) = log\frac{1+n_d}{1+df(d,t)} + 1$$
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html]: ``\log``, also ``tf` and ``tf-idf`` etc should be in ``\text{}``

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 5]
```diff
@@ -0,0 +1,1020 @@
...
+
+# Character n-grams
+
+---
+#Principle
+<br>
+<br>
+
+.center[
+![:scale 100%](images/do_you_want_ants.png)
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html:739]: hm this was an animation with a window of size 3 sliding over the text...  I realize it's not in the PDF... do you think you could get it from the odp? we need to have 4 different images here, I guess...

File: slides/aml-19-040418-working-with-text-data/index.html [hunk 6]
```diff
@@ -0,0 +1,1020 @@
...
+```
+164632
+```
+```python
+lr_char = LogisticRegressionCV().fit(X_train_char, y_train_sub)
+X_val_char = char_vect.transform(text_val)
+lr_char.score(X_val_char, y_val)
+```
+```
+0.88112000000000001
```
Comment (amueller) [inline slides/aml-19-040418-working-with-text-data/index.html]: round please

**Comment by amueller** [comment]
mostly looks good. it would be cool if it would be possible to get the missing animation...

**Comment by rajathkmp** [comment]
Wasn't able to get the animation image from the odp file, file's behaving strangely on a mac.

**Comment by amueller** [comment]
thanks, I'll fix it.

**Pull request merged**